Assets/MRTK/Core/Inspectors/HoverLightInspector.cs

```diff
@@ -18,5 +18,19 @@ private Bounds OnGetFrameBounds()
             Debug.Assert(light != null);
             return new Bounds(light.transform.position, Vector3.one * light.Radius);
         }
+
+        [MenuItem("GameObject/Light/Hover Light")]
+        private static void CreateHoverLight(MenuCommand menuCommand)
```

Comment:
This may be better suited for future PR but wanted to call out. 

Can you render default inspector but add the documentation button for hoverlight and proximity light to each component. Like this doc button:

![image](https://user-images.githubusercontent.com/25975362/76574509-c2307f00-6479-11ea-9802-d8d095a4ae08.png)

Reply:
Just curious, does that button do something different than the built-in documentation button above it?